index.js

```diff
@@ -0,0 +1,91 @@
+/***********************************
+ * DOM-Elements
+ **********************************/
+let form = $('form');
+let inputFrom = $('input[name="from"]');
+let inputTo = $('input[name="to"]');
+let from = $("#from");
+let to = $("#to");
+let result = $(".result");
+let error = $(".error");
+
+/***********************************
+ * Methodes
+ **********************************/
+function getResult(){
+    inputTo.val(calculateResult(inputFrom.val(),from.val(), to.val()));
+}
+
+function calculateResult(value, from, to){
+
+    value = parseFloat(value);
+    //celsius to Kelvin
+    if(from === 'c' && to === 'k'){
```

Comment:
Consider using additional parentheses for better readability.

Reply:
```suggestion
    if((from === 'c') && (to === 'k')){
```